translations/ui-tenant-settings/en_GB.json

```diff
@@ -130,7 +130,7 @@
     "settings.pluginNames.find-erm-usage-data-provider": "Find ERM usage metadata provider",
     "settings.pluginNames.find-finc-metadata-source": "Find finance metadata source",
     "permission.module.enabled": "UI: Tenant-settings module is enabled",
-    "permission.settings.enabled": "Settings (tenant): display list of settings pages",
+    "permission.settings.enabled": "Settings (tenant): View",
     "permission.settings.addresses": "Settings (tenant): Can manage tenant addresses",
```

Comment:
Sorry for the question, but should we make any changes to `en_GB.json` and similar files?

Reply:
@SergiyVSergiyenko I made them directly here because sometimes the Lokalise service we use for translations is delayed.